rust/tool/src/install.rs

```diff
@@ -84,9 +87,9 @@ impl Installer {
                 .rev()
                 .collect()
         };
-        self.install_generations_from_links(&links)?;
 
         self.install_systemd_boot()?;
+        self.install_generations_from_links(&links)?;
```

Comment:
I'm not sure if this would change behaviour. Probably better to leave it as it was.

Reply:
Right, it was a mistake.